<html><h3>:x: ci:test - 0 out of 1 jobs passed in 19 minutes 44 seconds 218 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/4475c8b5440b6b259440f04109ebca9e022e8078">4475c8b5440b6b259440f04109ebca9e022e8078</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/1/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-6.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/1/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/1/">test-subrepository-acceptance-pullrequest(7.1.x)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>prepare-portal-repository:
     [exec] Warning: Permanently added 'test-1-0,172.16.0.50' (RSA) to the list of known hosts.
     [exec] Warning: Permanently added 'test-1-0,172.16.0.50' (RSA) to the list of known hosts.
     [exec] rsync: read errors mapping "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/244/git-hash": No such device or address (6)
     [exec] WARNING: git-hash failed verification -- update discarded (will try again).
     [exec] rsync: read errors mapping "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/244/git-hash": No such device or address (6)
     [exec] WARNING: git-hash failed verification -- update discarded (will try again).
     [exec] git-hash
     [exec] git-hash
     [exec] 
     [exec] sent 68 bytes  received 527 bytes  0.66 bytes/sec
     [exec] total size is 41  speedup is 0.07
     [exec] rsync: read errors mapping "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/244/git-hash": No such device or address (6)
     [exec] ERROR: git-hash failed verification -- update discarded.
     [exec] rsync: read errors mapping "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/244/git-hash": No such device or address (6)
     [exec] ERROR: git-hash failed verification -- update discarded.
     [exec] rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1637) [generator=3.1.1]
     [exec] rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1637) [generator=3.1.1]
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20180801072322540

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-subrepository-acceptance-pullrequest.xml:257: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-subrepository-acceptance-pullrequest.xml:170: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:695: exec returned: 23</code></pre></div></li></ol></details></html>

--- a/commerce-product-test/src/testFunctional/macros/CPCommerceCatalog.macro
+++ b/commerce-product-test/src/testFunctional/macros/CPCommerceCatalog.macro
@@ -71,7 +71,7 @@
 		<if>
 			<or>
 				<equals arg1="${optionFieldType}" arg2="checkbox-multiple" />
-				<equals arg1="${optionFieldType}" arg2="Select" />
+				<equals arg1="${optionFieldType}" arg2="Select from List" />
 				<equals arg1="${optionFieldType}" arg2="Radio" />
 			</or>
 			<then>

--- a/commerce-product-test/src/testFunctional/macros/CommerceConfiguration.macro
+++ b/commerce-product-test/src/testFunctional/macros/CommerceConfiguration.macro
@@ -5,7 +5,7 @@
 		</execute>
 
 		<execute macro="CommerceNavigator#gotoCommercePortletNavItem">
-			<var name="navItem" value="Health Status" />
+			<var name="navItem" value="Health Check" />
 		</execute>
 
 		<for list="1,2,3,4,5" param="buttonRowNumber">

--- a/commerce-product-test/src/testFunctional/paths/CommerceEntry.path
+++ b/commerce-product-test/src/testFunctional/paths/CommerceEntry.path
@@ -128,6 +128,12 @@
 	<td>//div/input[contains(@id,'numericISOCode')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TITLE_FIELD</td>
+	<td>//div/input[contains(@id,'title')]</td>
+	<td></td>
+</tr>
+<tr>
 	<td>NEVER_EXPIRE_CHECKBOX</td>
 	<td>//input[contains(@name,'neverExpire') and (@type='checkbox')]</td>
 	<td></td>

